System/Vector.h

```diff
@@ -543,6 +545,9 @@ namespace RTE {
 		/// Clears all the member variables of this Vector, effectively resetting the members of this abstraction level only.
 		/// </summary>
 		void Clear() { m_X = m_Y = 0; }
+
+		float m_Tol = 0.0000001F; //!< Floating point comparison tolerance.
```

Comment:
Given this member variable isn't commonly used, I'd rather a more verbose name for it. To support nice doxygen output we add member comments and stuff, but I'd still rather that in cases like this the name describe the variable pretty clearly.
It doesn't help that in certain fonts l and 1 can be hard to distinguish at a glance so this looks kind of like m_To1 which is even weirder.

I'd expand it to `m_FloatComparisonTolerance` or `m_FloatTolerance`, or at least `m_Tolerance`.